applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -108,5 +108,13 @@ def _runTest(self,settings_file_name):
         potential_flow_analysis = PotentialFlowAnalysis(model, settings)
         potential_flow_analysis.Run()
 
+class Naca0012SmallTest(PotentialFlowTestFactory):
```

Comment:
Regarding the tests, they are not working in my machine. It seems there was some issue when merging. The tests should look something like this test [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MeshMovingApplication/tests/test_ale_fluid_solver.py). 

So, your SmallLiftJumpTest should be defined inside the PotentialFlowTests class as a function from it. Something like this:
```
class PotentialFlowTests(UnitTest.TestCase):

...

    def test_SmallLiftJumpTest(self):
        file_name = "small_lift_jump"
        settings_file_name = file_name + "_parameters.json"
        work_folder = "naca0012_small_test"

        with WorkFolderScope(work_folder):
            self._runTest(settings_file_name)

            for file_name in os.listdir():
                if file_name.endswith(".time"):
                    kratos_utilities.DeleteFileIfExisting(file_name)
```